packages/react-app/src/components/modals/optout-modal.tsx

```diff
@@ -160,7 +160,12 @@ export default function OptoutModal({ questData, onClose = noop }: Props) {
               icon={<GiCrossedSwords />}
               className="open-unplay-button"
               onClick={() => setOpened(true)}
-              label="Opt out"
+              label={
+                questData.creatorAddress === walletAddress &&
```

Comment:
There is computing here that will be called in each refresh tick, create a label memo that is recomputed when  either questData.creatorAddress, walletAddress or questData.players change